### DISCUSSION TITLE
Adaptive Card

### DISCUSSION THREAD

**Discussion by m3d0c** [microsoft-search/pnp-modern-search#2981] [Q&A] [answered]
Hi, 
I have a SharePoint list which is showing tools for the organisation. I am trying to create a search page which show these elements. The search page shows the items on a page but the when I click on the icon, it show the list item. I would like to replace this link with a url field in the list item. I can see this field in the debug view but when I try to set the aref in the adaptive card to the column name, it just shows nothing and I get a JSON string where the title was shown.

I have tried to follow the template tutorial and that doesnt work for me either.

I am not a developer, so therefore this is all new territory for me 

Any help would be appreciated

Dhiran

**Comment by wobba** [comment, marked as answer]
Can you try using html/handlebar templates instead of an adaptive card? Documentation at https://microsoft-search.github.io/pnp-modern-search/extensibility/templating/#reference-a-slot-in-a-adaptive-card explains how to reference a field in an adaptive card.

And feel free to share parts of your adaptive card layout on how you have configured it.

**Reply by m3d0c** [reply]
@wobba Thank  you for that. I also contacted you via Twitter and found that I could actually use the handlebars/HTML template to achieve my goal.
